modelhub/tests_modelhub/functional/modelhub/test_model_hub_funnel_discovery.py

```diff
@@ -126,6 +125,30 @@ def test_get_navigation_paths(db_params):
         use_to_pandas=True,
     )
 
+    # test different location stack column by name and different type
+    bts = funnel.get_navigation_paths(by='user_id', data=df, steps=2, location_stack='session_hit_number')
```

Comment:
```suggestion
    bts = funnel.get_navigation_paths(data=df, steps=2, location_stack='session_hit_number')
```

`by='user_id'` should be removed otherwise, we have to change the expected data.